### PR TITLE
[Do no merge] Outro data

### PR DESCRIPTION
**Pull request by jamesgorrie**
* Adds "Try these next" heading to outro
  screen shot 2018-10-18 at 12 20 43

* Adds the ability to add the outro data in Prismic
  screen shot 2018-10-18 at 13 14 05

Next up - adding it to the page

### CODE REVIEW

File: common/services/prismic/articles.js [hunk 1]
```diff
@@ -200,7 +281,7 @@ export async function getArticles(req: ?Request, {
     Prismic.Predicates.any('document.type', ['articles', 'webcomics'])
   ].concat(predicates), {
     orderings,
-    graphQuery,
+    graphQuery: graphQuery,
```
Comment (davidpmccormick) [inline common/services/prismic/articles.js]: What no shortcut? Just curious.

File: content/webapp/pages/article.js [hunk 2]
```diff
@@ -194,6 +194,7 @@ export class ArticlePage extends Component<Props, State> {
       }
     }).filter(Boolean);
 
+    console.info(article.outroResearchTitle);
```
Comment (davidpmccormick) [inline content/webapp/pages/article.js]: is this staying?

File: common/services/prismic/articles.js (hunk 1 above)
Comment (jamesgorrie) [reply common/services/prismic/articles.js]: It was just as I was trying to replace some of the whitespace characters to give us more room in the URL (Prismic sends the Graph query on the URL for caching reasons).

It was moved to here:
https://github.com/wellcometrust/wellcomecollection.org/pull/3566/files#diff-dccecb3db8753d09dbf462271a9a8ba5R169

File: content/webapp/pages/article.js (hunk 2 above)
Comment (jamesgorrie) [reply content/webapp/pages/article.js]: Nope

**Comment by jamesgorrie** [comment]
Happy with the code to be reviewed, just can't merge as there is an issue with Prismic not updating the model with fields.

**Comment by jamesgorrie** [comment]
Closing for #3566

**Pull request closed without merging**